### PR TITLE
Feature/standardized font size and layout fixes

### LINKED ISSUE #115: Make font sizes uniform

### PR DESCRIPTION
**Pull request by Frigyes06**
Fixes #115

### CODE REVIEW

File: src/gui/views/list.rs [hunk 1]
```diff
@@ -516,15 +517,15 @@ impl List {
                         .width(17)
                         .horizontal_alignment(alignment::Horizontal::Center)
                         .style(style::Text::Commentary)
-                        .size(17),
+                        .size(16),
```
Comment (adhirajsinghchauhan) [inline src/gui/views/list.rs:520]: Would be better to set `default_text_size: 16.0` in mod.rs, and remove all `size(16)`s.

File: src/gui/views/list.rs [hunk 2]
```diff
@@ -516,15 +517,15 @@ impl List {
                         .width(17)
```
Comment (adhirajsinghchauhan) [inline src/gui/views/list.rs:517]: nit: I'd suggest increasing icon widths to make the button containers appear more square; looks better.

22 should do it (also in navigation_menu.rs).

**Review by adhirajsinghchauhan — CHANGES_REQUESTED** [review]

**Review by adhirajsinghchauhan — APPROVED** [review]

**Comment by Frigyes06** [comment]
image
These texts get a bit cut off

**Comment by shapeshifterincontext** [comment]
Can we go back to the font and size used in this. V1.0.3 is not 'bad eyesight' friendly. The letters in each word are squashed together.

**Comment by Rudxain** [comment]
> Can we go back to the font and size used in this.

I agree, the text is too small

**Comment by adhirajsinghchauhan** [comment]
Could you please post screenshots? We didn't change text sizes again after this PR.

**Comment by Rudxain** [comment]
`cargo run` on `main`:
Screenshot  2024-09-05 03-14-54

- Linux 6.10
- Debian 13 "Trixie"
- GNOME 46
- Wayland 1.23 (+ Mutter)
- Intel HD Graphics 4000
- Display resolution = 1366×768 px
- GNOME display scale = 1x (hidden from GUI, because display is not "hi-DPI")
- Tweaks "Font" size scale factor = 1.5

**Comment by AnonymousWP** [comment]
For me it doesn't look small, neither does the text on your screenshot.

**Comment by Rudxain** [comment]
The review modal text isn't small, but it is relative to this
img

Although the pack list seems even smaller to me

**Comment by AnonymousWP** [comment]
> The review modal text isn't small, but it is relative to this img
> 
> Although the main pack list (the one behind the modal) seems even smaller to me

Your screenshot returns a 404. It's a private user URL.

**Comment by Rudxain** [comment]
> 404

That's odd 👀😕. It's the same screenshot [from here](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/pull/118#issuecomment-1868602553). Both load from my POV

**Comment by adhirajsinghchauhan** [comment]
> > 404
> 
> That's odd 👀😕. It's the same screenshot [from here](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/pull/118#issuecomment-1868602553). Both load from my POV

Notice the `jwt` parameter in that URL, though. It would differ per session, so what works for you won't work for others. Not sure if this token could be used to gain access to your account, but I'd contact GitHub anyway, to scrub your comment.

**Comment by Rudxain** [comment]
> `jwt`

[It seems to have an expiration date](https://github.com/orgs/community/discussions/109034#discussioncomment-9545999)

So there's not much to worry about. I'll replace the screenshot anyways

**Comment by Rudxain** [comment]
Now that I'm using a hi-DPI (1080p) display, I've set scale=2 in the GNOME Settings. The widgets are bigger!

**Pull request merged**